src/servicebus/Elsa.AzureServiceBus/Activities/ServiceBusSend.cs

```diff
@@ -0,0 +1,85 @@
+﻿using Elsa;
+using Elsa.Attributes;
+using Elsa.Design;
+using Elsa.Expressions;
+using Elsa.Results;
+using Elsa.Services;
+using Elsa.Services.Models;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Linq;
+using Elsa.AzureServiceBus.Services;
+
+namespace Elsa.AzureServiceBus.Activities
+{
+    [ActivityDefinition(
+        Category = "Azure",
+        Description = "Send a message via azure service bus",
+        RuntimeDescription = "x =>  x.state.queue ? `Send <strong></strong> to <strong>${x.state.queue}</strong>` :  x.definition.description)",
+        //Icon = "fas fa-user-plus",
+        Outcomes = new[] { OutcomeNames.Done })]
+
```

Comment:
Minor; please remove unnecessary blanks. I'm seeing this in many of the files. I won't comment on every instance, but please do remove unnecessary blank lines everywhere. This also goes for lines containing variable declarations and initializations (except for multi-line statements - they must be surrounded by blanks lines).

Thanks!